bundle/compliance/policy/gcp/sql/ensure_db_flag_is_as_expected.rego

```diff
@@ -5,10 +5,11 @@ import future.keywords.every
 import future.keywords.if
 import future.keywords.in
 
-is_flag_enabled(flag_name) if {
+is_flag_configured_as_expected(flag_name, expected_vals) if {
```

Comment:
consider renaming to indicate that not all expected values need to be present, only at least 1 of them